upgraderesponder/service.go

```diff
@@ -146,6 +160,13 @@ func (s *Server) initDB() error {
 	if err := s.createDB(InfluxDBDatabase); err != nil {
 		return err
 	}
+	// Uncomment this block if we support changing the InfluxDBContinuousQueryPeriod value by commandline argument
+	//if err := s.dropContinuousQueries(InfluxDBDatabase); err != nil {
+	//	return err
+	//}
+	if err := s.createContinuousQueries(InfluxDBDatabase); err != nil {
```

Comment:
There can be multiple pods running the same image in the deployment. Will this be handled gracefully?

Reply:
Yes, it will be handled gracefully. If users don't `InfluxDBContinuousQueryPeriod`, `s.createContinuousQueries()` is the same as NoOP.